### PR TITLE
This is the way

### PR DESCRIPTION
**Pull request by leeKol**
Hi, I'd love for your review.

### CODE REVIEW

File: this_is_the_way.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import glob
+import os
+
+
```
Comment (itayday) [inline this_is_the_way.py:4]: Great job!

File: this_is_the_way.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+def get_deep_files(path):
+    """
+    The function gets a path to a directory, and returns a list of all the files
+    that start with "deep" in the directory.
+    :param path: The path of the directory we want to extract the information from.
+    :return: A list of all the files that start with "deep" in the directory.
+    """
+    return [os.path.basename(file) for file in glob.glob(path + "/deep*")]
```
Comment (talbarfm) [inline this_is_the_way.py]: Nice! glob is my favorite directory to get paths with

File: this_is_the_way.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import glob
+import os
+
+
+def get_deep_files(path):
```
Comment (talbarfm) [inline this_is_the_way.py]: Use type hints and return types

File: this_is_the_way.py (hunk 2 above)
Comment (talbarfm) [inline this_is_the_way.py]: You have a small bug here - what If I had a directory names deepurple? whould it it return? is it a file?

**Review by talbarfm — CHANGES_REQUESTED** [review]

**Review by lovodkin93 — APPROVED** [review]

**Pull request merged**